config/site_settings.yml

```diff
@@ -580,7 +580,7 @@ files:
     client: true
     shadowed_by_global: true
   external_system_avatars_url:
-    default: "https://avatars.discourse.org/letter/{first_letter}/{color}/{size}.png"
+    default: "//{hostname}/letter_avatar_proxy/letter/{first_letter}/{color}/{size}.png"
```

Comment:
Just use `/letter_avatar_proxy/letter/{first_letter}/{color}/{size}.png`, no need to include the hostname.